hyacinth/session.py

```diff
@@ -54,29 +63,50 @@ def wrapper(self, *args, **kwargs):
 
 
 class Session:
-    """Session class for interacting with Clio Manage API.
+    """
+    Session class for interacting with Clio Manage API.
 
-        WARNING: enabling `ratelimit` will block the process synchronously
-        when API rate limits are hit. Support for async hyacinth is coming
-        soon.
-    :
+    WARNING: enabling `ratelimit` will block the process
+    synchronously when API rate limits are hit. Partial support
+    for async hyacinth is provided by hyacinth/async_session.py".
     """
 
     def __init__(
-        self,
-        token,
-        client_id,
-        client_secret,
-        ratelimit=False,
-        raise_for_status=False,
-        update_token=lambda *args: None,  # default update_token does nothing
+            self,
+            token,
+            client_id,
+            client_secret,
+            region="US",
+            ratelimit=False,
+            raise_for_status=False,
+            update_token=lambda *args: None,  # default update_token does nothing
     ):
-        """Initialize Session with optional ratelimits."""
+        """Initialize Clio API HTTP Session."""
+        if region == "US":
+            self.base_url = CLIO_BASE_URL_US
+            self.api_base_url = CLIO_API_BASE_URL_US
+        elif region == "CA":
+            self.base_url = CLIO_BASE_URL_CA
+            self.api_base_url = CLIO_API_BASE_URL_CA
+        elif region == "AU":
+            self.base_url = CLIO_BASE_URL_AU
+            self.api_base_url = CLIO_API_BASE_URL_AU
+        elif region == "EU":
+            self.base_url = CLIO_BASE_URL_EU
+            self.api_base_url = CLIO_API_BASE_URL_EU
+        else:
+            log.warning(f"Invalid region supplied: {region}, defaulting to 'US'")
```

Comment:
Do we want to default to the US or just throw an error and leave it at that? If the user & account in question is on a different geo instance, it'll fail if we try to make a request with their token on the US instance. Right?

Reply:
This is if they provide a region like "JP" or "CN", one that Clio has no regional support for

Reply:
If the token doesn't make the provided region, this session's requests will fail

Reply:
Ok sounds good. Makes sense since `US` is the default regardless